src/runtime_src/core/include/xrt_mem.h

```diff
@@ -37,7 +37,7 @@ extern "C" {
 #define	XCL_BO_FLAGS_DEV_ONLY		(1 << 28)
 #define	XCL_BO_FLAGS_HOST_ONLY		(1 << 29)
 #define	XCL_BO_FLAGS_P2P		(1 << 30)
-#define	XCL_BO_FLAGS_EXECBUF		(1 << 31)
+#define	XCL_BO_FLAGS_EXECBUF		(unsigned int)(1 << 31)
```

Comment:
This does not seem right place for cast,  where is the compile failure?

Reply:
Thanks. Since we are using C++14, I've now changed these to `const uint32_t`